shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Deleg.hs

```diff
@@ -222,6 +232,7 @@ delegationTransition = do
       -- note that pattern match is used instead of regCred, as in the spec
       -- hk ∉ dom (_stkCreds ds) -- Specification code translates below
       not (haskey hk (_stkCreds ds)) ?! StakeKeyAlreadyRegisteredDELEG hk
+      not (haskey (RewardAcnt network hk) (_rewards ds)) ?! StakeKeyInRewardsDELEG hk
```

Comment:
want to add this one link change to the spec too? I can do it if that would be helpful.

Reply:
I think this change isn't too important (it's a sanity check), but if you felt like adding the `applyRUpd` changes to the spec that would be great!